src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1334,6 +1345,38 @@ static ucs_status_t uct_dc_mlx5_calc_sq_length(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
+static void
+uct_dc_mlx5_dci_set_port_affinity(uct_dc_mlx5_iface_t *iface,
+                                  const uct_dc_mlx5_iface_config_t *config)
+{
+    uct_ib_mlx5_md_t *md = ucs_derived_of(iface->super.super.super.super.md,
+                                          uct_ib_mlx5_md_t);
+
+    iface->tx.port_affinity = 0;
+    if (md->super.dev.port_select_mode == UCT_IB_LAG_QUEUE_AFFINITY ||
+        md->super.dev.port_select_mode == UCT_IB_LAG_PORT_SELECT_FT) {
+        switch (config->tx_port_affinity) {
+        case UCS_CONFIG_ON:
+            iface->tx.port_affinity = 1;
+            break;
+        case UCS_CONFIG_AUTO:
+            if (md->super.dev.port_select_mode == UCT_IB_LAG_QUEUE_AFFINITY) {
+                iface->tx.port_affinity = 1;
+            }
+            break;
+        default:
+            break;
+        }
```

Comment:
need to simplify this code using if/else along with comments

Reply:
Done